samples/msgext-search-quickstart/js/README.md

```diff
@@ -11,96 +11,102 @@ languages:
 extensions:
  contentType: samples
  createdDate: "07/07/2021 01:38:27 PM"
-urlFragment: officedev-microsoft-teams-samples-msgext-search-quickstart-js
+urlFragment: officedev-microsoft-teams-samples-msgext-action-quickstart-js
```

Comment:
This sample is related to msgext-search as per sample name.?

Reply:
yes